.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: java
-jdk: openjdk8
+jdk:
+  - openjdk8
+  - openjdk11
```

Comment:
IIRC we automatically push artifacts on merge to develop, so this would cause artifacts to be pushed twice.

Maybe extend https://github.com/syncany/syncany/tree/develop/gradle/upload scripts to only upload one of the builds? 

I would personally also be fine with just using jdk11

Reply:
I'm ok with moving to jdk11.

Reply:
Another piece to fix is to switch the doclet (APIviz doesn't have a Java 11 version).  Either the standard, or maybe [Asciidoclet](https://asciidoctor.org/docs/asciidoclet/).

Reply:
I don't know why we had gone with APIviz previously. The javadoc ends up being hosted here: https://docs.syncany.org/javadoc/ Otherwise I don't think we have any uses for it, so any alternative that renders it okay is fine for me.